modules/issue_tracker/php/edit.class.inc

```diff
@@ -458,7 +458,7 @@ class Edit extends \NDB_Page implements ETagCalculator
 
         // Attachment for new issue.
         $uploadedFiles = $request->getUploadedFiles();
-        if (isset($uploadedFiles)) {
+        if (isset($uploadedFiles) && !empty($uploadedFiles)) {
```

Comment:
```suggestion
        if (!empty($uploadedFiles)) {
```

the isset is unnecessary. empty already suppresses unset errors. (And it's always set, it's declared on the line above)